helm-charts/common/mm-embedding/README.md

```diff
@@ -0,0 +1,58 @@
+# OPEA mm-embedding microservice
+
+Helm chart for deploying OPEA multimodal embedding service.
+
+## Installing the Chart
+
+To install the chart, run the following:
+
+```console
+cd GenAIInfra/helm-charts/common
+export MODELDIR=/mnt/opea-models
+export HFTOKEN="insert-your-huggingface-token-here"
+# To deploy embedding-multimodal-bridgetower microserice on CPU
+helm install mm-embedding mm-embedding --set global.modelUseHostPath=${MODELDIR} --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN}
+# To deploy embedding-multimodal-bridgetower microserice on Gaudi
+# helm install mm-embedding mm-embedding --set global.modelUseHostPath=${MODELDIR} --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --values mm-embedding/gaudi-values.yaml
+# To deploy embedding-multimodal-clip microserice on CPU
+helm install mm-embedding mm-embedding --set global.modelUseHostPath=${MODELDIR} --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --values mm-embedding/variant_clip-values.yaml
+```
+
+By default, the embedding-multimodal-bridgetower service will downloading the "BridgeTower/bridgetower-large-itm-mlm-itc" download which is about 3.5GB, and the embedding-multimodal-clip service will download the "openai/clip-vit-base-patch32" model which is about 1.7GB.
```

Comment:
'By default, the embedding-multimodal-bridgetower service will downloading ' -> '... will download'
'"BridgeTower/bridgetower-large-itm-mlm-itc" download' -> '"BridgeTower/bridgetower-large-itm-mlm-itc" model'

Reply:
done